services/twitter/twitter.service.js

```diff
@@ -116,49 +94,31 @@ class TwitterFollow extends LegacyService {
     }
   }
 
-  static registerLegacyRouteHandler({ camp, cache }) {
-    camp.route(
-      /^\/twitter\/follow\/@?([^/]+)\.(svg|png|gif|jpg|json)$/,
-      cache((data, match, sendBadge, request) => {
-        const user = match[1] // eg, shields_io
-        const format = match[2]
-        const options = {
-          url: `http://cdn.syndication.twimg.com/widgets/followbutton/info.json?screen_names=${user}`,
-        }
-        const badgeData = getBadgeData(`follow @${user}`, data)
+  async fetch({ user }) {
+    return this._requestJson({
+      schema,
+      url: `http://cdn.syndication.twimg.com/widgets/followbutton/info.json?screen_names=${user}`,
```

Comment:
Should `user` be URL encoded? We may want to specify it as a query string parameter like we do in the [Wercker service](https://github.com/badges/shields/blob/f0e037e0404c40271f9e72c49cdd12d5e33d53fc/services/wercker/wercker.service.js#L31) for instance.